documentation/bolt_installing_modules.md

```diff
@@ -329,6 +329,27 @@ module-install:
     proxy: https://forge-proxy.com:8080
 ```
 
+## Install modules using an authorization token
```

Comment:
Hm, maybe worth mentioning that R10K and Puppet configuration is ignored, so you must include this configuration when using Bolt to install modules that require authorization.

Reply:
I lean towards not mentioning r10k or Puppet config. This section (and the previous two) already states that if you need to use an auth token to install modules you need to configure Bolt. Adding that you need to do this in Bolt because we don't load r10k or Puppet config seems redundant.